nengo_interfaces/tests/test_changing_airsim_settings.py

```diff
@@ -0,0 +1,85 @@
+# NOTE tests if the airsim interface warns the user if a setting.json parameter changes and requires
+# a UE4 sim restart
+# make sure that the Capture Settings: Width parameter does not match what you current have in your
+# ~/Documents/Airsim/settings.json
+import json
+import math
+from os.path import expanduser
+
+import nengo
+import numpy as np
+
+from nengo_interfaces.airsim import AirSim
+
+# read the current capture width so we can:
+# - change it to test if we get warned, change it back
+# - test that we do not get an exception if it doesn't change
+home = expanduser("~")
+with open(  # pylint: disable=W1514
+    f"{home}/Documents/AirSim/settings.json", "r+"
+) as fp:
+    data = json.load(fp)
+    prev_width = data["CameraDefaults"]["CaptureSettings"][0]["Width"]
+
+
+# Test begins here
+airsim_dt = 0.01
+steps = 500
+
+green = "\033[92m"
+red = "\033[91m"
+endc = "\033[0m"
+
+# change a value in settings.json, this should throw a RuntimeError
+try:
+    interface = AirSim(
+        dt=airsim_dt,
+        camera_params={
+            "use_physics": True,
+            "fps": 1,
+            "save_name": "hi",
```

Comment:
Since there are no images being saved to this folder, this can be set to `None` to not create the folder.